roles/create_day2_cluster/tasks/main.yml

```diff
@@ -1,3 +1,11 @@
+---
+- name: Deprecation warning
```

Comment:
Does Ansible do some magic to detect this as a deprecation warning? Or should this have a when clause?

Reply:
I was wrongly assuming that the magic is triggered by setting the deprecated attribute in the metadata. I found that we need to check thes status of ansible.builtin.configm, so I updated the notice task accordingly